core/src/main/java/software/amazon/awssdk/core/http/loader/DefaultSdkAsyncHttpClientBuilder.java

```diff
@@ -34,12 +33,12 @@
             ));
 
     @Override
-    public SdkAsyncHttpClient createHttpClientWithDefaults(AttributeMap serviceDefaults) {
-        // TODO We create and SdkHttpClientFactory every time. Do we want to cache it instead of the service binding?
+    public SdkAsyncHttpClient buildWithDefaults(AttributeMap serviceDefaults) {
+        // TODO We create and build every time. Do we want to cache it instead of the service binding?
```

Comment:
I believe the original comment was supposed to be 'we create a SdkHttpClientFactory'